psutil/arch/windows/ntextapi.h

```diff
@@ -551,6 +552,26 @@ DWORD (CALLBACK *_GetActiveProcessorCount) (
 
 #define GetActiveProcessorCount _GetActiveProcessorCount
 
+BOOL(CALLBACK *_WTSQuerySessionInformation) (
+    HANDLE hServer,
+    DWORD SessionId,
+    WTS_INFO_CLASS WTSInfoClass,
+    LPTSTR* ppBuffer,
+    DWORD* pBytesReturned
+    );
+
+BOOL(CALLBACK *_WTSEnumerateSessions)(
+    HANDLE hServer,
+    DWORD Reserved,
+    DWORD Version,
+    PWTS_SESSION_INFO* ppSessionInfo,
+    DWORD* pCount
+    );
+
+VOID(CALLBACK *_WTSFreeMemory)(
+    IN PVOID pMemory
+    );
+
```

Comment:
Since these names are available on other Windows systems, I think there should be something like:

```c
#ifdef WINDOWS_NANO_OR_SOMETHING
...
#endif 
```

There's no need of the `_` prefix. Look at how other declarations are done. E.g.:

```c
PSTR (NTAPI * _RtlIpv6AddressToStringA) (
    struct in6_addr *Addr,
    PSTR P);

#define RtlIpv6AddressToStringA _RtlIpv6AddressToStringA
```

Reply:
> I'm not entirely sure I understand what you mean

It's OK, let me try to rephrase it differently.
What I am aiming at is using the original API names, without the `_` or `psutil_` prefix (the original version of this file did exactly that, then I changed it). 
In order to do that you first have to "declare" the API name, like this:
https://github.com/giampaolo/psutil/blob/96cc7ea40d6f4f08e86677434213cef119cf1748/psutil/arch/windows/ntextapi.h#L501-L507
Then you'll have to "load" the API, in here, like this:
https://github.com/giampaolo/psutil/blob/96cc7ea40d6f4f08e86677434213cef119cf1748/psutil/_psutil_common.c#L217-L220
Is it more clear?

Reply:
Not tested, but it should be something like this:

```c
// ntextapi.h
BOOL (CALLBACK *_WTSQuerySessionInformation) (
    HANDLE hServer,
    DWORD SessionId,
    WTS_INFO_CLASS WTSInfoClass,
    LPTSTR* ppBuffer,
    DWORD* pBytesReturned
    );

#define WTSQuerySessionInformation _WTSQuerySessionInformation
```

```c
// _psutil_common.c
WTSEnumerateSessions = psutil_GetProcAddressFromLib(
    "wtsapi32.dll", "WTSEnumerateSessions");
if (! WTSEnumerateSessions)
    return 1;
```

Reply:
Also, MS doc says:
> The wtsapi32.h header defines WTSEnumerateSessions as an alias which automatically selects the ANSI or Unicode version of this function based on the definition of the UNICODE preprocessor constant. 

...so I think you should keep using `WTSEnumerateSessions` instead of `WTSEnumerateSessionsW`.

Reply:
If you can fix the issue just by adding that new header (or from setup.py) there is no need to redefine anything in ntextapi.h.

Just define them explicitly in ntextapi.h only if you can’t solve it otherwise, in which case you should put the definition inside an “#ifdef nanoserver” because I think it will break other windows versions (but the CI tests should break)